styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -518,19 +518,19 @@ private synchronized void tryPollPods() {
 
     final RunState runState = stateManager.get(workflowInstance);
     if (runState == null) {
-      LOG.warn("Pod event for unknown or inactive workflow instance {}", workflowInstance);
+      LOG.debug("Pod event for unknown or inactive workflow instance {}", workflowInstance);
```

Comment:
This one doesn't happen nearly as often, does it? I think I'd keep this one and let the others be debug.

Reply:
This happens quite frequently actually, at least every time a workflow instance becomes not active.